### DISCUSSION TITLE
Data storage issues!

### DISCUSSION THREAD

**Discussion by mieyeaa** [anyproto/contributors#128] [Q&A]
1 - Data is stored locally in the selected path not.
2- There is no switch for data upload synchronization, or is it too hidden?
3--Because it is installed on the C drive by default. There is no option to install the path, if the data is saved locally, it is also saved in the C drive, and if it is used for a long time, it will definitely lead to the bloat of the C drive.

**Comment by fuksman** [comment]
Hello @mieyeaa,

You can select a local-only mode, then your data will not be backed up to the external infrastructure: https://doc.anytype.io/anytype-docs/data-and-security/self-hosting#local-only-mode

The ability to choose a custom storage location will be introduced later: https://github.com/orgs/anyproto/projects/1?pane=issue&itemId=29227794
